bcda/auth/plugin/alpha.go

```diff
@@ -154,13 +156,24 @@ func (p *AlphaAuthPlugin) ValidateAccessToken(token string) error {
 }
 
 func (p *AlphaAuthPlugin) DecodeAccessToken(token string) (jwt.Token, error) {
-	return jwt.Token{}, errors.New("Not yet implemented")
+	t, err := jwt.Parse(token, func(token *jwt.Token) (interface{}, error) {
+		if _, ok := token.Method.(*jwt.SigningMethodRSA); !ok {
+			return nil, fmt.Errorf("Unexpected signing method: %v", token.Header["alg"])
```

Comment:
is there a better way to handle/log this error?

Reply:
Probably down to personal preference, but I think this error handling makes sense, with it bubbling up and being returned from `DecodeAccessToken()` later. The calling function can decide how to handle it and whether it should be logged.